src/dapps/preview/preview-options.ts

```diff
@@ -27,6 +27,7 @@ export type PreviewOptions = {
   wheelZoom?: number | null
   wheelPrecision?: number | null
   wheelStart?: number | null
-  transparentBackground?: boolean
+  background?: string | null
```

Comment:
I'm not entirely sure by the description if this property refers to the background color.What is the value of this property? Can we add a JSDoc here explaining what it is?
What if we use the backgroundColor name as well?